cassandra/src/main/java/com/yahoo/ycsb/db/CassandraCQLClient.java

```diff
@@ -85,6 +72,24 @@
   public static final String READ_TIMEOUT_MILLIS_PROPERTY =
       "cassandra.readtimeoutmillis";
 
+  private static List<String> fieldList;
```

Comment:
Actually, maybe they should be. All threads with an instance of this class should share them. These variables are declared in the `synchronized` block.